site/styles/_footer.scss

```diff
@@ -21,14 +21,19 @@ footer {
 
 .footer-nav {
   margin-bottom: 30px;
+  vertical-align: middle;
```

Comment:
This doesn't appear to have any effect... probably delete it?

Reply:
fixed